src/api/compatibility/application/CompatibleApplicationResourceTreeItem.ts

```diff
@@ -107,13 +107,19 @@ type Resolvable<T> = T & {
 export function createResolvableProxy<T extends AzExtParentTreeItem>(resolvable: Resolvable<T>): T {
     const providerHandler: ProxyHandler<Resolvable<T>> = {
         get: (target: Resolvable<T>, name: string): unknown => {
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
```

Comment:
```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'ResolvedAppResourceBase'.
  No index signature with a parameter of type 'string' was found on type 'ResolvedAppResourceBase'.
ts(7053)
```

The errors are super nasty and not easily bypassed with casting. It's a result of using the proxy. I added a comment to one of them to explain.